test-suite/README.md

```diff
@@ -1,13 +1,63 @@
-#Local
+# Local
 
-### Installing
-
-Run the shell script for local development:
+## Running Test Harness
+### Pre-Requisites:
+* Install Java 17
+* Install Docker
+* Install a preferred IDE
 
+### Installing
+1. Run Docker Desktop
+2. Navigate to project directory via IDE Terminal
+3. cd into `docker` folder
+4. run command: 
+```shell
+#For Windows
+./start-local-environment.sh
+
+#For Mac or Linux use
+./start-local-environment-arm.sh)
```

Comment:
This script has been removed in #266. Can run the same `./start-local-environment.sh`.